app/views/shared/_accept_cookies_banner.html.erb

```diff
@@ -1,26 +1,33 @@
-<nav class="navbar navbar-default navbar-fixed-bottom accept_cookies_banner_nav">
-    <div id="accept_cookies_banner_container" class="container-fluid">
-        <div class="row"
-            role="dialog"
-            aria-live="polite"
-            aria-label="accept_cookies"
-            aria-describedby="accept_cookies_copy" >
-            <div class="col-sm-10 i-accept-copy" id="accept_cookies_copy">
-                This website uses cookies to perform analytics,
-                improve functionality, and enhance user experience.
-                You may set your browser to
-                <a href="https://www.wikihow.com/Disable-Cookies">decline cookies</a>,
-                but some content on our site may not display properly.
-                For more information, please visit our
-                <a href="https://www.sciencehistory.org/site-terms">privacy policy</a>.
-            </div>
-            <div class="col-sm-2 i-accept-div text-center">
-                <a href="#"
-                    class="i-accept-link"
-                    aria-label="dismiss cookie message"
-                    role="button"
-                    tabindex="0">I Accept</a>
+<%
+    if Rails.application.config.respond_to?(:hide_accept_cookies_banner) &&
+    Rails.application.config.hide_accept_cookies_banner
+%>
+    <%# Don't show this cookie banner to the unit tests unless they specifically request it.%>
+<% else %>
+    <nav class="navbar navbar-default navbar-fixed-bottom accept_cookies_banner_nav">
```

Comment:
no need for an `if` with an empty body followed by `else` -- just use `unless` or `if !`.